guardian/support-frontend
# Newspaper archive - SSR custom header

## What are you doing in this PR?

Creating a new server side rendered page to eventually be iframed on to theguardian.newspapers.com as the custom header.



[**Trello Card**](https://trello.com/c/pr57zkkZ/1099-customise-newspaper-archivecom-header)

## Why are you doing this?

Currently on there is a header that exists as an interactive. We want to create a new version of this, and have it easier to maintain, so we are hosting it on the support site.



## How to test

Run `yarn build-ssr` if locally 
Go to `/newspaper-archive-header` and see the hello world SSR rendered page at the moment.



## Screenshots
![image](https://github.com/user-attachments/assets/f801550d-3f0b-47ed-9366-fd95e6f0bf44)

## Next steps: 
Combine the two scripts into one, this would make it easier to add SSR rendered content in future without duplicating the `tsx` scripts.
Develop the actual content of the header.
